src/app/nav/nav.component.html

```diff
@@ -85,10 +85,9 @@ <h3>WHARF</h3>
       <div pTooltip='This feature has not yet been implemented. Stay tuned!' tooltipPosition="bottom">
         <p-menu styleClass="menu-item" [model]="items"></p-menu>
       </div>
+      <p-menu styleClass="menu-item" *ngIf="!isUsingPlaceholderOidcConfig" [model]="loginItem"></p-menu>
       <p-menu styleClass="menu-item" [model]="documentationItem"></p-menu>
     </div>
-
-    <p-menu styleClass="menu-item" [model]="userItem" pTooltip='This feature has not yet been implemented. Stay tuned!'
-      tooltipPosition="top"></p-menu>
+    <p-menu styleClass="menu-item" *ngIf="!isUsingPlaceholderOidcConfig" [model]="userItem"></p-menu>
```

Comment:
`isUsingPlaceholderOidcConfig` is a method, so it only checks the truthyness of the method being a value. You seem to be missing a `| async` pipe here.

Suggest inverting the semantic of it as well so there's no need for negation here in the HTML code. Ex:

```html
<p-menu styleClass="menu-item" *ngIf="shouldShowUserItem$()|async" [model]="userItem"></p-menu>
```

```ts
  public shouldShowUserItem$(): Observable<boolean> {
    return this.configService.isPlaceHolderOidcConfig$
      .pipe(map(x => !x));
  }
```

Reply:
Was reverted. Resolved.